src/transactions/feed/TransactionDetailsScreen.tsx

```diff
@@ -100,12 +106,16 @@ function TransactionDetailsScreen({ route }: Props) {
       } else {
         content = <TransferReceivedContent transfer={receivedTransfer} />
       }
-
       break
     case TokenTransactionTypeV2.SwapTransaction:
       content = <SwapContent exchange={transaction as TokenExchange} />
       retryHandler = () => navigate(Screens.SwapScreenWithBack)
       break
+    case TokenTransactionTypeV2.EarnClaimReward:
+    case TokenTransactionTypeV2.EarnWithdraw:
+    case TokenTransactionTypeV2.EarnDeposit:
+      content = null //TODO details for Earn
```

Comment:
is there a separate ticket for this?

Reply:
There is not. I'll make one, but we are still lacking designs.

Reply:
Added ticket and a link to it in 21cfc10839664eebf27f38bf1d4863ad11f8f939!